ci/lsan-suppressions.txt

```diff
@@ -1,2 +1,8 @@
 # Xrandr is leaking...
 leak:libXrandr
+
+# libfontconfig too according to
+# https://chromium.googlesource.com/chromium/src/build/+/master/sanitizers/lsan_suppressions.cc#22
+# http://crbug.com/39050
+# https://www.reddit.com/r/cpp/comments/41vm7r/is_clangs_leak_sanitizer_inaccurate_or_is_qt/
+leak:libfontconfig
```

Comment:
Still seems to make problems with the CI, can you try `leak:libfontconfig.so.1` and see if that fixes it?

Reply:
The lsan-suppression is fine, it is just that one test is failing, because it doesn't get the LSAN_OPTIONS env passed.
I'll add the fix here.

Reply:
Great, thanks!